BDArmory/Damage/HitpointTracker.cs

```diff
@@ -1557,6 +1568,31 @@ void SetHullMass()
             }
             _hullConfigured = true;
         }
+        private void CalculateRCSreduction()
+        {
+            float radarReflected = armorRadarReturnFactor < 10 ? 1 - (Mathf.Log(Mathf.Max(ArmorThickness, 1f), 10) * (1 - armorRadarReturnFactor)) : //armor < 10 will have reduced radar absorbsion
```

Comment:
Using `Mathf.Log10(x)` is faster than `Mathf.Log(x, 10)` (0.00818µs vs 0.0132µs) due to `Log(10)` being a predefined constant.